packages/parts/src/lib/maps/RegionModal.svelte

```diff
@@ -78,6 +79,8 @@
           lat={centerLat}
           {zoom}
           on:click={({ detail: { lng, lat } }) => add({ longitude: lng, latitude: lat })}>
+          <slot />
```

Comment:
Note how using a slot makes us able to repurpose an already existing marker component and also keeps us from having to pass in the whole dictionary object and add types and such - it's a lot cleaner.

Reply:
Got it! Thanks